src/libostree/ostree-repo-pull.c

```diff
@@ -914,6 +981,9 @@ meta_fetch_on_complete (GObject           *object,
   pull_data->n_outstanding_metadata_fetches--;
   pull_data->n_fetched_metadata++;
   check_outstanding_requests_handle_error (pull_data, local_error);
+
```

Comment:
Added a newline here. Don't care if it stays or goes, but just wanted to make sure you were aware.